engines/cpu.c

```diff
@@ -89,12 +263,27 @@ static int fio_cpuio_init(struct thread_data *td)
 
 	o->nr_files = o->open_files = 1;
 
-	log_info("%s: ioengine=%s, cpuload=%u, cpucycle=%u\n",
-		td->o.name, td->io_ops->name, co->cpuload, co->cpucycle);
+	switch (co->cpumode) {
```

Comment:
This one also needs breaking up like the previous one.

Reply:
And probably the default case belongs here, so we catch it at init time as we should. Avoids the check in the runtime/fast path.

Reply:
yup and fix the last one too. Sorry.